### DISCUSSION TITLE
quickly adding context in chat

### DISCUSSION THREAD

**Discussion by gotoolkits** [codefuse-ai/codefuse-ide#39] [Ideas]
Is it possible to use the Continue-dev plugin? Or implement similar functions like @file, @dir, @Codebase, @URL, @Docs for quickly adding context
![image](https://github.com/user-attachments/assets/e51d92ae-2b50-4987-9dfe-259d32f643e0)

**Comment by hacke2** [comment]
Very good suggestion, but compared to Chat, we are focusing more on code generation this year, similar to Cursor on code generation core functionality:
1. line level: cursor tab
2. fast level: inline chat
3. current file level: instant apply
4. multiple file level: cursor composer

This year OpenSumi core will focus on 1 and 2, 3 and 4 may be next year's focus, but of course we welcome community PRs so that we can get 3 and 4 done ahead of schedule!
